src/ru/nsu/ccfit/zuev/osu/Config.java

```diff
@@ -67,6 +67,8 @@ public class Config {
         forceRomanized,
         fixFrameOffset,
         removeSliderLock,
+        removeNoteLock,
+        removeNoteLockWhenMiss,
```

Comment:
Allocating those variables on Config class doesn't match the porpouse of the class, it only storage fields that are saved in SharedPreferences.